pkg/envoy/types_test.go

```diff
@@ -0,0 +1,15 @@
+package envoy
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+)
+
+func TestTypeURIString(t *testing.T) {
+	assert := tassert.New(t)
+
+	expected := "test-content"
```

Comment:
For more complex tests where you want to test against multiple inputs, table driven testing is a great approach. Refer to https://github.com/openservicemesh/osm/blob/main/pkg/service/types_test.go#L46 for an example.

This particular test is trivial enough to not warrant a table.